source/developers/cook-books/how-tos/web-app-manifest-use-credentials.rst

```diff
@@ -0,0 +1,57 @@
+:is-up-to-date: True
+
+.. index:: Enable Use of Cookies and Credentials by the Web App Manifest, Web App Manifest, Manifest
+
+===================================================
+Enabling Use of Credentials by the Web App Manifest
+===================================================
+
+Some sites uses a web app manifest, a file that describes the site.
+
+From `Google <https://developers.google.com/web/fundamentals/web-app-manifest>`_:
+
+   The web app manifest is a simple JSON file that tells the browser about your web application and how it should behave when ‘installed’ on the user’s mobile device or desktop.
+
+To tell the browser about the manifest, a link tag is added to all pages of your site:
+
+.. code-block:: html
+
+   <link rel="manifest" href="/manifest.json">
+
+|
+
+To allow the web app manifest to use cookies and credentials, ``crossOrigin="use-credentials"`` must be specified in the manifest tag.
```

Comment:
Might be good to make allusion to the fact that Crafter CMS requires cookies to be sent for site selection?